src/oidcendpoint/authn_event.py

```diff
@@ -24,30 +25,36 @@ def expires_in(self):
         return self["valid_until"] - time_sans_frac()
 
 
-def create_authn_event(uid, salt, authn_info=None, **kwargs):
+def create_authn_event(uid, authn_info=None, authn_time: int = 0,
+                       valid_until: int = 0, expires_in: int = 0, **kwargs):
     """
 
+    :param authn_time:
     :param uid:
-    :param salt:
     :param authn_info:
+    :param valid_until:
+    :param expires_in:
     :param kwargs:
```

Comment:
Fix params, not `uid` param. Move `authn_info` to start:
```
:param authn_info
:param authn_time
:param valid_until
:param expires_in
```

Reply:
I think this is already in there. At least I have it in my copy. So no problem with this.

Reply:
UID is used further down. Why do you say it's not  needed ?